index.src.html

```diff
@@ -656,6 +656,31 @@ <h2 id="integration-with-html">Integration with HTML</h2>
 </section>
 
 <section>
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  The CSS Standard does not specify how it fetches resources referenced from
+  stylesheets. However, implementations use the URL a given stylesheet was
```

Comment:
Is this section informative or normative?  Because if this is meant to be normative, then "the URL a given stylesheet was loaded from" needs to be defined much more clearly than it is now.  If this is meant to be informative, the text is still confusing, of course...